_source/_plugins/stackblitz.rb

```diff
@@ -0,0 +1,35 @@
+module Jekyll
+    class StackBlitzTag < Liquid::Tag
+        def initialize(name, id, tokens)
+            super
+            @id = parse_id(id)
+            @file = parse_file(id)
+        end
+
+        def render(context)
+            %(<p>
```

Comment:
Have you tried rendering this in a test post? The reason I ask is because you don't always get the positioning and padding desired with simple HTML. See @deepu105's PR (https://github.com/oktadev/okta-blog/pull/965) for some suggestions I added.

Reply:
Another thing to check: does it render nicely on a mobile device? We've had to add `style="max-width: 100%" to iframes in the past.

Reply:
Yep, I checked rendering on a post, but I didn't try different form factors. I'll double check.

Reply:
Desktop
![Screen Shot 2021-11-18 at 12 16 16 PM](https://user-images.githubusercontent.com/7586237/142473542-1b2471fd-f740-4f66-9183-23403e372d36.png)

Mobile (400px wide)
![Screen Shot 2021-11-18 at 12 17 08 PM](https://user-images.githubusercontent.com/7586237/142473604-fecb5621-d8ea-4425-907b-39a9d7da4828.png)

Reply:
@mraible i can add more vertical space, but it doesn't look too squished to me rn either...

Reply:
@mraible since it's in a paragraph tag, it looks like it's consistent margin-bottom. i also applied max-width style just in case and it looks the same as the pic above too.